File: padaria-app/src/app/module/models/src/employee/gender.ts
import { describe } from '@module/utils/functions/enum';

export enum Gender {
  None = 0,
  Male = 1,
  Female = 2,
  Other = 3,
}

describe(Gender, {
  None: 'Nenhum',
  Male: 'Masculino',
  Female: 'Feminino',
  Other: 'Outro',
});

File: padaria-api/src/main/java/com/padaria/model/gender/Gender.java
package com.padaria.model.gender;

public enum Gender {

    Male("Masculino"),
    Female("Feminino");

    private final String description;

    Gender(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}
